<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dean_flow</class>
 <widget class="QWidget" name="dean_flow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="text_De">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_down">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>439</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Down">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_up">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Параметры</string>
   </property>
  </widget>
  <widget class="Line" name="line_up">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>439</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="Clear_calculation">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="flow_De_calculation">
      <property name="text">
       <string>Расчёт De</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" rowspan="5">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_p">
        <property name="text">
         <string>Плотность жидкости (p)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_D">
        <property name="text">
         <string>Диметр (D)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_v">
        <property name="text">
         <string>Скорость течения (v)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_nu">
        <property name="text">
         <string>Динамическая вязкость (μ)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Rc">
        <property name="text">
         <string>Радиус кривизны пути (r)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="5">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit_Rc"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_D"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_nu"/>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_p"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_v"/>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_kg">
      <property name="text">
       <string>кг/м³</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_mm">
      <property name="text">
       <string>мм</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_m_c">
      <property name="text">
       <string>м/с</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_Pa_c">
      <property name="text">
       <string>Па · с</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_mm_2">
      <property name="text">
       <string>мм</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
